corehq/apps/export/models/new.py

```diff
@@ -1355,7 +1355,8 @@ def generate_schema_from_builds(cls, domain, app_id, identifier, force_rebuild=F
         app_build_ids.extend(cls._get_current_app_ids_for_domain(domain, app_id))
 
         for app_doc in iter_docs(Application.get_db(), app_build_ids, chunksize=10):
-            if app_doc.get('doc_type', '') != 'Application':
+            doc_type = app_doc.get('doc_type', '')
+            if not (doc_type == 'Application' or doc_type == 'LinkedApplication'):
```

Comment:
This would probably be clearer with `if doc_type not in ('Application, 'LinkedApplication')`